### DISCUSSION TITLE
RfC: Plan the phase out of django CMS 3.x

### DISCUSSION THREAD

**Discussion by macolo** [django-cms/django-cms#8053] [General]
- We have django CMS 4.x now and want to focus development efforts there
- 3.x has LTS until 2026
- we should have a plan for communication around the fact that at some point in time we will cease to provide LTS and releases for 3.x

**Comment by fsbraun** [comment]
I agree: We have been candid about the situation, but we should have a continuous communication plan of some sorts to not leave people alone. In particular, I think, **testimonials of successful transitions** should be helpful.
* https://www.django-cms.org/en/blog/2023/12/26/upgrade-to-django-cms-4/
* https://docs.django-cms.org/en/latest/#software-version-requirements-and-release-notes
